### PR TITLE
Adding support to PT-BR.

### PR DESCRIPTION
**Pull request by Skellington**
Translation of Telegram D to Portuguese (Brazil).
This is the first version. A first revision is coming soon.

### CODE REVIEW

**Comment by Aokromes** [comment]
Telegram already have PTBR language, maybe this PR is no longer needed?

File: Telegram/Resources/lang-pt_br.txt [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+lng_settings_workmode_tray: "Mostrar ícone na bandeja";
+lng_settings_workmode_window: "Mostrar ícone na barra de tarefas";
+lng_settings_auto_start: "Iniciar o Telegram quando junto ao sistema";
+lng_settings_start_min: "Iniciar minimizado";
+lng_settings_add_sendto: "Place Telegram in Â«Send toÂ» menu";
+lng_settings_scale_label: "Escala de interface";
+lng_settings_scale_auto: "Automático ({cur})";
+
+lng_settings_section_chat: "Opções de bate-papo";
+lng_settings_replace_emojis: "Replace emojis";
```
Comment (TiagoDanin) [inline Telegram/Resources/lang-pt_br.txt]: Line 175: Missed putting in Portuguese here
The translation would be: "Substituir emojis"

**Comment by TiagoDanin** [comment]
Line 82: Translation: "Grupo é inacessível"
Line 149: Translation: "Mostrar nome de que enviou"
Line 170: Translation: "Coloque o telegrama no menu «Enviar para»"
Line 192: Translation: "Pasta personalizada, apenas manualmente"
Line 249: Translation: "Remover"
Line 249: Translation: "Remover {user} do grupo?"
Line 294: Translation: "{from} removeu {user}"

Withdraws "Â»" and "Â«", exchange for "»" and "«"
Line with "Â»": 105, 170, 190, 254, 255, 257, 258, 260, 261, 263, 264, 283, 301, 302, 391
Line with Â«: 170, 190, 283, 301, 302, 391

**Comment by TiagoDanin** [comment]
Line 125: Translation: "Erro na imagem selecionada."
Line 144: Translation: "Selecione a área quadrado para sua foto do perfil"
Line 456: Translation: "Foto Individual"

**Comment by telegramdesktop** [comment]
Languages are not managed from here, sorry.

**Pull request closed without merging**